mlrun/config.py

```diff
@@ -467,6 +467,30 @@
         "expose_internal_api_endpoints": False,
     },
     "default_workflow_runner_name": "workflow-runner-{}",
+    "log_collector": {
+        "address": "localhost:8282",
+        # log collection mode can be one of: "sidecar", "legacy", "best-effort"
+        # "sidecar" - use the sidecar to collect logs
+        # "legacy" - use the legacy log collection method (logs are collected straight from the pod)
+        # "best-effort" - use the sidecar, but if for some reason it's not available use the legacy method
+        # note that this mode also effects the log querying method as well, meaning if the mode is "best-effort"
+        # the log query will try to use the sidecar first and if it's not available it will use the legacy method
+        # TODO: once this is changed to "sidecar" by default, also change in common_fixtures.py
+        "mode": "legacy",
```

Comment:
Need to change to sidecar

Reply:
the opposite i think. retain as legacy for when running under docker